### PR TITLE
docs: Liferay Theme Tasks update Additional Theme Dependencies blurb

### PR DESCRIPTION
**Pull request by pat270**

### CODE REVIEW

File: packages/liferay-theme-tasks/README.md [hunk 1]
```diff
@@ -254,10 +254,31 @@ Version of Liferay Portal this theme is intended for.
...
+Get the latest `liferay-frontend-theme-unstyled`:
+
+```
+npm i -DE liferay-frontend-theme-unstyled@latest
+```
+
+Get the latest `liferay-frontend-theme-styled`:
+
+```
+npm i -DE liferay-frontend-theme-styled@latest
```
Comment (wincent) [inline packages/liferay-theme-tasks/README.md:272]: To make this less obfuscated, I'd use long-form for these `npm` invocations:

- `i` → `install`
- `-D` → `--save-dev`
- `-E` → `--save-exact`

File: packages/liferay-theme-tasks/README.md [hunk 2]
```diff
@@ -254,10 +254,31 @@ Version of Liferay Portal this theme is intended for.
 
 ## Additional Theme Dependencies
 
-In order for themes to successfully build, they must declare additional dependencies in their package.json file such as [liferay-frontend-theme-unstyled](https://www.npmjs.com/package/liferay-frontend-theme-unstyled) and [liferay-frontend-theme-styled](https://www.npmjs.com/package/liferay-frontend-theme-styled).
+In order for themes to successfully build, they must declare additional dependencies in their `package.json` file such as [liferay-frontend-theme-unstyled](https://www.npmjs.com/package/liferay-frontend-theme-unstyled) and [liferay-frontend-theme-styled](https://www.npmjs.com/package/liferay-frontend-theme-styled).
 
-These additional dependencies are normally included automatically in themes created with the [Theme Generator](https://github.com/liferay/generator-liferay-theme).
+If you would like to require specific versions of these dependencies, you can specify them directly in your theme's `package.json` file and remember to `npm install` afterwards.
```
Comment (wincent) [inline packages/liferay-theme-tasks/README.md:259]: Instead of "you can ... and remember", can we reword this as:

> these dependencies, specify them directly in your theme's `package.json` file and run `npm install` afterwards.

Because otherwise it sounds like an observation ("you can remember") rather than an instruction.

**Review by wincent — APPROVED** [review]
Looks good to me, but I have a couple of small tweaks suggested inline.

**Comment by pat270** [comment]
@wincent done!

**Comment by wincent** [comment]
lg2m!
